.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    "env": {
+        "es6": true,
+        "node": true
```

Comment:
this is not node code is it?

Reply:
Nope

Reply:
guess I got confused on what is node and what is browser.

Reply:
No worries -- when it comes to bedrock, the general rule is: if it says `bedrock-web-*` or `bedrock-vue-*` or `bedrock-angular-*` it's a browser thing. Otherwise it's a node thing. I think we have one unfortunately exception to this rule with `bedrock-web-push` which is a node thing.